Extensions/SendTo/ActionViewController.swift

```diff
@@ -26,15 +26,15 @@ class ActionViewController: UIViewController, ClientPickerViewControllerDelegate
             let instructionsViewController = InstructionsViewController()
             instructionsViewController.delegate = self
             let navigationController = UINavigationController(rootViewController: instructionsViewController)
-            presentViewController(navigationController, animated: false, completion: nil)
+            present(navigationController, animated: false, completion: nil)
             return
         }
 
         ExtensionUtils.extractSharedItemFromExtensionContext(self.extensionContext, completionHandler: { (item, error) -> Void in
-            guard let item = item where error == nil && item.isShareable else {
-                let alert = UIAlertController(title: Strings.SendToErrorTitle, message: Strings.SendToErrorMessage, preferredStyle: .Alert)
-                alert.addAction(UIAlertAction(title: Strings.SendToErrorOKButton, style: .Default) { _ in self.finish() })
-                self.presentViewController(alert, animated: true, completion: nil)
+            guard let item = item, error == nil && item.isShareable else {
```

Comment:
we don't need the and here, we could use
`guard let item = item, error == nil, item.isShareable`